src/sentry/static/sentry/app/utils/withUsersTeams.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import {Team, Organization} from 'app/types';
+import getDisplayName from 'app/utils/getDisplayName';
+import {Client} from 'app/api';
+
+// We require these props when using this HOC
+type DependentProps = {
+  api: Client;
+  organization: Organization;
+};
+
+type InjectedTeamsProps = {
+  teams: Team[];
+  loadingTeams: boolean;
+};
+
+type State = {
```

Comment:
Could this be

```
type State = InjectedTeamsProps & {error: Error|null}
```

Reply:
Yea that makes sense state should probably actually just resemble the InjectedTeamsProps and I can move error into InjectedTeamsProps